tessellation/src/path_stroke.rs

```diff
@@ -660,6 +660,44 @@ impl<'l, Output: 'l + GeometryBuilder<Vertex>> StrokeBuilder<'l, Output> {
                 }
             }
 
+            LineJoin::Bevel => {
+            
+                // First we take our initial vertex for compute our triangle
+                let current_vertex = add_vertex!(
+                    self,
+                    Vertex {
+                        position: self.current,
+                        normal: normal,
+                        advancement: self.length,
+                        side: front_side,
+                        }
+                );
+
+                // Compute 2 normal on the right and left side  
+                let normal = if front_side == Side::Right {
+                    vec2(a_line.y, -a_line.x).normalize() * 0.5
+                } else {
+                    vec2(-a_line.y, a_line.x).normalize() * 0.5
+                };
+
+                // Now we can add vextex at the end of our normals
+                let mut last_vertex = add_vertex!(
+                    self,
+                    Vertex {
+                        position: self.current,
+                        normal: normal,
+                        advancement: self.length,
+                        side: front_side,
+                        }
+                );
+                let back_vertex = last_vertex;
+                
+                self.output.add_triangle(back_vertex, current_vertex, last_vertex);
+
+                (back_vertex, last_vertex)
```

Comment:
Another thing that changed in #159 is that `self.prev_normal` has to be updated to the normal of the last or back vertex (depending on which side they are) by the code that handles the join, so you'll have to do something similar (see the new round join code).